blocks/filesystem.py

```diff
@@ -147,7 +146,23 @@ def ls(self, path):
             output = glob.glob(path)
         return sorted(p.rstrip('/') for p in output)
 
-    def copy(self, sources, dest):
+    def rm(self, paths):
```

Comment:
Is it the case that `rm` will do the same thing that `rm -rf` does in Unix-like systems, i.e. delete a full directory without prompting?  If so, given that this diverges from the standard usage of `rm` that people are used to, would it make sense to include a prompt or warning of some kind?

Reply:
It runs like `rm -r`, but not with the `-f`. It's a little safer. Do you think it makes sense to have `recursive` as an argument?

Reply:
I think having `recursive` as an argument might be more intuitive for new users since it's more similar to the `rm` usage for Unix-like systems, but don't have a strong opinion on this.

Reply:
Will add! I think that makes sense